parachain/src/chain_spec/interlay.rs

```diff
@@ -172,40 +172,41 @@ fn interlay_mainnet_genesis(
             disable_inclusion_check: false,
         },
         issue: interlay_runtime::IssueConfig {
-            issue_period: interlay_runtime::DAYS,
+            issue_period: interlay_runtime::DAYS * 2,
```

Comment:
In all likelyhood [the open refendum](https://interlay.subsquare.io/democracy/referendum/15) will be approved so let's keep this 1 day?